test/test.c

```diff
@@ -143,7 +143,14 @@ int test_case(const char *name, int xmssmt, unsigned int num_tests){
 int main()
 {
     int rc;
-    rc = test_case("XMSSMT-SHA2_22/2_192", 0, 1<<16);
+    rc = test_case("XMSS-SHA2_10_256", 0, 1<<11);
+    // rc |= test_case("XMSS-SHA2_16_256", 0, 1<<5);
```

Comment:
Can you clean up the lines that are commented out?

Reply:
Oh, I left these lines for testing, if uncomment all, it will perform a test for all RFC8391 parameters, take >20 minutes to finish the test, so I comment some parameter so the test can be finished in 5 mins.

Reply:
Eventually it would be good if the test suite lets one specify the individual algorithm to be tested. And because of the long key generation cost, perhaps we could have a mode that skips key generation by having a pre-generated key and only does signature and verification testing. But that's for later.

Reply:
You're right. I will write test from generated key via reading a file in the next pull request.